k9mail/src/main/java/security/planck/ui/passphrase/PassphraseDialogMode.kt

```diff
@@ -1,5 +1,5 @@
 package security.planck.ui.passphrase
 
 enum class PassphraseDialogMode {
-    MANAGE, UNLOCK
+    MANAGE, UNLOCK,
```

Comment:
do we need the extra , without any other option in the enum?

Reply:
Removed, thanks.